ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableDataProvider.kt

```diff
@@ -28,7 +30,17 @@ internal class VariableDataProvider(
     }
 
     fun periodName(rcPackage: Package): String {
-        return "PERIOD_NAME"
+        val stringId = when (rcPackage.packageType) {
+            PackageType.LIFETIME -> R.string.lifetime
+            PackageType.ANNUAL -> R.string.annual
+            PackageType.SIX_MONTH -> R.string.semester
+            PackageType.THREE_MONTH -> R.string.quarter
+            PackageType.TWO_MONTH -> R.string.bimonthly
+            PackageType.MONTHLY -> R.string.monthly
+            PackageType.WEEKLY -> R.string.weekly
+            PackageType.UNKNOWN, PackageType.CUSTOM -> null
+        }
+        return stringId?.let { applicationContext.getString(it) } ?: ""
```

Comment:
I'm defaulting to an empty string, which I think is the same we do in iOS.

Reply:
👍🏻